include/circt/Dialect/SV/SVVisitors.h

```diff
@@ -29,7 +29,7 @@ class Visitor {
         .template Case<
             // Expressions
             ReadInOutOp, ArrayIndexInOutOp, VerbatimExprOp, VerbatimExprSEOp,
-            ConstantXOp, ConstantZOp,
+            ConstantXOp, ConstantZOp, LocalparamOp,
```

Comment:
please put this in the "declarations" section next to RegOp, not in expressions.